### ISSUE TITLE
No events are imported

### ISSUE THREAD

**Issue by Pverte** [cloud-atlas-ai/obsidian-ics#107]
Hi Peeeople !
I've discovered this plugin this morning and tried to use it with Dataview, but I'm blocked on a small issue : ICS isn't importing any events.
I have tried with my school's Ical which is working on Google Calendar and my personal Google Calendar, but nothing change : When I do /ICS:Import Events, nothing is displayed, even on the logs ! All I can see is that the plugin day-planner gets an empty character in my daily note.
What could be the issue ?

**Comment by Pverte** [comment]
Is there any way to be sure that the plugin is gettings calendar events ?

**Comment by Pverte** [comment]
Okay, I've found out what is the issue : 
Spoiler alert : it's because of how is made the ical link of my school!

So first I looked simply with some console.log where could be the issue, with my school's generated link. And then I found that that the code will stop completely at the line `icsArray = parseIcs(await request({ url: calendarSetting.icsUrl }));`. So it's surely with the request ! I then told myself "Hey, how does curl sees the request ?" : HTML PAGE WITH A 301 ERROR

Here is how looks the link : 
`www.[school_link].fr/api/public/student/planning/[A random string]`. As on my browser it was directly downloading the ical, I thought that the ical was hidden *(yes, I was dumb)*.
I then decide to try a curl -L to the link and then I HAVE THE ICAL !

I'm a bit (completely) new to TypeScript, how can we wait that the request handle the redirections ? It seems that the request function in the plugin doesn't handle it.
And we should add a Notice to tell the user if there is an issue with the link (because here, the plugin stops without any notice).
